src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String rawData = readFile(fromFileName);
+        String evaluatedData = evaluateData(rawData);
+        writeFile(toFileName, evaluatedData);
+    }
+
+    public String readFile(String fileName) {
+        try (BufferedReader bufferedReader =
+                     Files.newBufferedReader(Paths.get(fileName))) {
+            String value = bufferedReader.readLine();
+            StringBuilder stringBuilder = new StringBuilder();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = bufferedReader.readLine();
+            }
+            return stringBuilder.toString();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't correctly read data from file " + fileName, e);
+        }
+    }
+
+    public String evaluateData(String data) {
```

Comment:
Pay attention to access modifiers, they should not always be public (check this in other cases)